README.md

```diff
@@ -1 +1,202 @@
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+ - any code editor
+ - intalled Node.js
+
+
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/Fernando426/Hello-world.git
+
+Example commands:
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+
+### Install
+
+Install this project with:
+- any code editor
+- Node.js
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="Fernando Montes"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@Fernando426](https://github.com/Fernando426)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [Fernando Montes](https://www.linkedin.com/in/fernando-jose-montes-linares-523822270/?originalSubdomain=ve)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<!-- Feel free to check the [issues page](../../issues/). -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+If you like this project, I would like to invite you to show your support for the growth and development of professional Web programming, any contribution, comment or suggestion will be very well received!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to first thank the team and the platform as such at Microverse for making this fun, enjoyable and useful phase of a whole path of preparation towards world-class professionalism and training. also to the people, coding partners, guides and everyone who in one way or another has contributed to the advancement of the entire program. thank you so much!!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Nice LICENSE 📄 but the link is currently broken ❌ Make sure to create a LICENSE file named `LICENSE` to fix this link  🔗 follow the instructions in the next line to know how to do that (once you're done remove the instructions as you won't need it anymore) 🙅🏽  take a look at what I get when I try to access the LICENSE for now 👇🏽 

![Screenshot 2023-04-26 at 20 51 17](https://user-images.githubusercontent.com/45739963/234732315-d95f7664-d5f6-48a7-a7a3-b7e64be78249.png)